### ISSUE TITLE
Unable to see remote video (occasionally)

### ISSUE THREAD

**Issue by qaz1991815** [ethand91/mediasoup-ios-client-sample#18]
Hello, I'm trying to connect to the official mediacup demo server, but I have a problem. There is a certain chance that I can't see the remote video when two IOS are talking. When I turn off the audio transmission, everything is normal, I don't know how to solve it, hope to get your help, thank you very much

**Comment by AndyCandy-max** [comment]
Hi qaz1991815, I tried to compile the code, however, I can only get the blank screen which has only four buttons. There is no place to enter the wss address. Can you help me what did I miss?

**Comment by qaz1991815** [comment]
You can modify the WSS address in the ViewController.swift，But if you want to communicate with the official mediasoup demo server, you need to change a lot

**Comment by AndyCandy-max** [comment]
Thank you @qaz1991815 , this exactly what I did, however, I have problem like this now: 

2021-02-04 19:53:50.388244-0500 mediasoup-ios-cient-sample[3651:91641] CFNetwork SSLHandshake failed (-9847)
2021-02-04 19:53:50.388420-0500 mediasoup-ios-cient-sample[3651:91641] TCP Conn 0x600002a1c160 SSLHandshake failed (-9847)
websocketDidDisconnect

Have you had the same issue before?

**Comment by qaz1991815** [comment]
I can share my source code for your reference, but it still has the above problem. If you can help me solve it, it will be a great joy for everyone. What's your email address?

**Comment by AndyCandy-max** [comment]
@zhang_qianyi@hotmail.com   So you have the same SSLHandshake failed issue?

**Comment by qaz1991815** [comment]
No, only have websocket disconnect issue

**Comment by samchen7910** [comment]
@qaz1991815 may I ask, how did you render the remote video track? I'm following the demo app and able to join the room, Chrome user is able to see and listen to iOS user, but iOS user is not able to listen or see (the view is black). I did receive the audio track and video track of the consumer via socket. (I attached demo code and my video track)
<img width="562" alt="Screenshot 2022-02-09 at 17 00 45" src="https://user-images.githubusercontent.com/24253311/153514657-51e0b495-32d6-49e9-88d1-efd650b274d2.png">
<img width="595" alt="Screenshot 2022-02-09 at 17 05 24" src="https://user-images.githubusercontent.com/24253311/153514735-a3f86bb5-fdf0-4ccd-85f1-bcb60f2b0e2c.png">
<img width="623" alt="Screenshot 2022-02-09 at 17 05 37" src="https://user-images.githubusercontent.com/24253311/153514737-a38a3fed-e66e-43ec-b936-be36d015bd2e.png">
